src/inline_spl_token.rs

```diff
@@ -0,0 +1,92 @@
+/// Partial SPL Token declarations inlined to avoid an external dependency on the spl-token crate
+/// Copied from solana-runtime
```

Comment:
Adding the dependency seems like the right call here

Reply:
Yes. I have created a PR to make this module public in solana-runtime. Once that is published and available I will remove from this